flexible-classroom/reference/restful-authentication.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Authenticate REST calls'
+sidebar_position: 9
+type: docs
+description: >
+  Links to the manual downloads for this product, and explanations on how to install them.
+---
+
+import DOC from '@docs/shared/common/_restful-authentication.mdx';
```

Comment:
Suggestion: Change the doc content from basic HTTP authentication to token authentication. You can refer to https://docs.agora.io /en/Real-time-Messaging/faq/restful_authentication to know about what's token authentication. 
Reason: The REST API authentication in Flexible Classroom is token authentication, not basic HTTP authentication.

Reply:
Done.